### PR TITLE
Add update manifest (fixes #53)

### LINKED ISSUE #53: Consider adding automatic updates
We added this to searchengine-devtools a while back:

https://github.com/mozilla-extensions/searchengine-devtools/commit/bfb86b490aa2130262e3b54e6d5d3123017438b8

It is a little bit more work to bump releases, but makes it easier to roll-out updates:

https://github.com/mozilla-extensions/searchengine-devtools/commit/af0632b4ab98ddfedbb8bf22cd876258675ff8b6

### PR DESCRIPTION
**Pull request by leplatrem**
fixes #53

### CODE REVIEW

File: update.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "addons": {
+    "remote-settings-devtools@mozilla.com": {
+      "updates": [
+        {
+          "version": "1.3.0",
+          "update_link": "https://github.com/mozilla-extensions/remote-settings-devtools/releases/download/1.3.0/remote-settings-devtools-1.3.0.xpi"
```
Comment (Standard8) [inline update.json:7]: Just to note this doesn't work at the moment, as the name of the xpi is different. However, I think if you start uploading direct from the builders, then you'll get the version number there by default, so it probably won't be an issue.

**Review by Standard8 — APPROVED** [review]
Looks good.

**Comment by Standard8** [comment]
Oh the other thing to note, is that generally I bump all the version numbers (and tag) before doing the release, then once the xpi is signed I just edit the release and add the xpi. The add-on update system copes with the missing xpi until it appears.

File: update.json (hunk 1 above)
Comment (leplatrem) [reply update.json:7]: I don't really how to upload direct from the builders :/ 

At least, I'd like to a proper release procedure with clear steps in the README. I will update the PR based on your comments

Comment (Standard8) [reply update.json:7]: Oh sorry that was confusing. I meant that you download it, and then upload it without renaming (I think the searchengine-devtools ones have the version in automatically).

And yes, I should document the process.

File: README.md [hunk 2]
```diff
@@ -33,3 +33,12 @@ Run in a browser with live-reload:
 ```
 web-ext run --firefox-binary ~/path/to/firefox -s ./extension/
 ```
+
+# Release
+
+1. Bump version in ``package.json``, ``update.json``, and ``extension/manifest.json``
+2. Tag commit ``git tag -a X.Y.Z`` and push it
+3. Create release with changelog on Github
```
Comment (Standard8) [inline README.md:41]: You shouldn't need to create the tab manually if you're creating a release on github - github should tag it for you.

**Review by Standard8 — APPROVED** [review]

**Pull request merged**